Automattic/wp-calypso
# Checkout: Add feature list for Jetpack Growth

Related to https://github.com/Automattic/jetpack-roadmap/issues/1957

## Proposed Changes

* add checkout feature list for Jetpack Growth

## Why are these changes being made?


* providing users with more information when purchasing new Jetpack Growth plan

## Testing Instructions



* apply the branch to you local host
* go to `/checkout/jetpack/jetpack_growth_yearly` on your local host
* verify that the features on the left side are populated and you see more than one feature

![SCR-20241115-ijpy](https://github.com/user-attachments/assets/84948b71-fafa-40db-aead-6d59023afd6f)

## Pre-merge Checklist
